events_nebula.xml.append

```diff
@@ -7377,14 +7377,14 @@ IDEAS
 
 <eventList name="NEBULA_HAZARD_EXCESSION_LIST">
 	<event>
-		<text>The magnetic field emitted by the object prevents you from immediately acquiring a single AI ship that survived the battle. They avatars where about to dispatch a probe towards the object, but then turn to engage you.</text>
+		<text>The magnetic field emitted by the object prevents you from immediately acquiring a single AI ship that survived the battle. They avatars were about to dispatch a probe towards the object, but then turn to engage you.</text>
```

Comment:
`<text>The magnetic field emitted by the object prevents you from immediately acquiring a single AI ship that survived the battle. The avatars were about to dispatch a probe towards the object, but then turn to engage you.</text>`

Change [They] with [The].